source/gui/settingsDialogs.py

```diff
@@ -792,8 +794,9 @@ def onSave(self):
 		config.conf["general"]["askToExit"]=self.askToExitCheckBox.IsChecked()
 		config.conf["general"]["playStartAndExitSounds"]=self.playStartAndExitSoundsCheckBox.IsChecked()
 		logLevel=self.LOG_LEVELS[self.logLevelList.GetSelection()][0]
-		config.conf["general"]["loggingLevel"]=logging.getLevelName(logLevel)
-		logHandler.setLogLevelFromConfig()
+		if not logHandler.isLogLevelForced:

```

Comment:
This will always be true. I think you meant ```isLogLevelForced()```

Reply:
Of course, sorry for the nasty typo.
Fixed as of 896297ff0c